apps/entity-service/src/entities/processors/project-report.processor.ts

```diff
@@ -0,0 +1,195 @@
+      }
+      builder.where({ projectId: project.id });
+    }
+
+    return { models: await builder.execute(), paginationTotal: await builder.paginationTotal() };
+  }
+
+  async addFullDto(document: DocumentBuilder, projectReport: ProjectReport) {
+    const projectReportId = projectReport.id;
+    const taskId = projectReport.taskId;
+    const reportTitle = await this.getReportTitle(projectReport);
+    const siteReportsIdsTaks = ProjectReport.siteReportIdsTaksSubquery([taskId]);
+    const seedsPlantedCount = (await Seeding.visible().siteReports(siteReportsIdsTaks).sum("amount")) ?? 0;
+    const treesPlantedCount =
+      (await TreeSpecies.visible().collection("tree-planted").siteReports(siteReportsIdsTaks).sum("amount")) ?? 0;
+    const approvedSiteReportsFromTask = await SiteReport.approved()
+      .reportsTask(taskId)
+      .findAll({ attributes: ["id", "siteId", "numTreesRegenerating"] });
+    const regeneratedTreesCount = sumBy(approvedSiteReportsFromTask, "numTreesRegenerating");
+    const siteReportsCount = await SiteReport.reportsTask(taskId).count();
+    const nurseryReportsCount = await NurseryReport.reportsTask(taskId).count();
+    const migrated = !!projectReport.oldModel;
+    const seedlingsGrown = await this.getSeedlingsGrown(projectReport);
+    const siteReportsUnsubmittedIdsTask = await ProjectReport.siteReportsUnsubmittedIdsTaskSubquery([taskId]);
+    const nonTreeTotal = (await Seeding.visible().siteReports(siteReportsUnsubmittedIdsTask).sum("amount")) ?? 0;
+    const readableCompletionStatus = await this.getReadableCompletionStatus(projectReport.completion);
+    const createdByUser = projectReport.user ? projectReport.user : null;
+    const props: AdditionalProjectReportFullProps = {
+      reportTitle,
+      seedsPlantedCount,
+      treesPlantedCount,
+      regeneratedTreesCount,
+      siteReportsCount,
+      nurseryReportsCount,
+      migrated,
+      seedlingsGrown,
+      nonTreeTotal,
+      readableCompletionStatus,
+      createdByUser,
+      ...(this.entitiesService.mapMediaCollection(
+        await Media.projectReport(projectReportId).findAll(),
+        ProjectReport.MEDIA
+      ) as ProjectReportMedia)
+    };
+    document.addData(projectReport.uuid, new ProjectReportFullDto(projectReport, props));
+  }
+
+  async addLightDto(document: DocumentBuilder, projectReport: ProjectReport) {
+    document.addData(projectReport.uuid, new ProjectReportLightDto(projectReport));
+  }
+
+  protected async getReportTitle(projectReport: ProjectReport) {
+    if (projectReport.frameworkKey == "ppc" || !projectReport.dueAt) {
+      return projectReport.title ?? "";
+    } else {
+      const dueAt = new Date(projectReport.dueAt);
+      dueAt.setMonth(dueAt.getMonth() - 1);
+      const wEnd = dueAt.toLocaleString("en-US", { month: "long", year: "numeric" });
+
+      dueAt.setMonth(dueAt.getMonth() - 5);
+      const wStart = dueAt.toLocaleString("en-US", { month: "long" });
```

Comment:
Is `en-US` what we're using for date formatting on the PHP side? I have a faint memory that in the FE, we typically prefer `en-GB` for date formatting when the user's locale isn't easily available (although we could fetch it here, and maybe that would be better?), but I'm not 100% sure about that.